FEXCore/Source/Interface/Core/JIT/JITClass.h

```diff
@@ -61,6 +62,15 @@ class Arm64JITCore final : public CPUBackend, public Arm64Emitter {
   const bool HostSupportsRPRES {};
   const bool HostSupportsAFP {};
 
+  struct RestartOptions {
+    FEXCore::LongJump::JumpBuf RestartJump;
+    bool NeedsLongJumps {};
+    enum class RestartOptionControl : uint64_t {
+      Incoming = 0,
+      NeedsLongJumps = 1,
```

Comment:
Probably better to rename this to `EnableLongJumps` (or `Enable<XYZ>` after renaming the `NeedsLongJumps` variable).

When seeing `RestartOptionControl::NeedsLongJumps`, I wouldn't have thought of reading it as `Restart *because* long jumps are needed`. It seems clearer to have it read as `Restart to enable long jumps` with the new name.